packages/admin-next/dashboard/src/routes/orders/order-create-fulfillment/components/order-create-fulfillment-form/order-create-fulfillment-item.tsx

```diff
@@ -79,10 +78,10 @@ export function OrderCreateFulfillmentItem({
               <Text className="txt-small" as="span" weight="plus">
                 {item.title}
               </Text>
-              {item.variant.sku && <span>({item.variant.sku})</span>}
+              {item.variant?.sku && <span>({item.variant.sku})</span>}
```

Comment:
It will be present, but our types dictate that it's possibly undefined. You know, back to the whole discussion around whether or not our default fields and relations should be types as undefined or not.

Reply:
This is also why I have to do the shenanigans on line 25.

Reply:
😂 yes I understand, but in that case wouldn't it make it cleared to use `!` all the way down instead of mixing? Cause that's the thing that made me think that it can really be undefined in the end